src/main/java/com/zakem/RecipeCatalog/object/Ingredient.java

```diff
@@ -0,0 +1,31 @@
+package com.zakem.RecipeCatalog.object;
```

Comment:
Does a database spin up when you run this? If not, you might want to add that with some pre-installed data.